code/modules/capture_the_flag/ctf_classes.dm

```diff
@@ -5,7 +5,6 @@
 	ears = /obj/item/radio/headset
 	uniform = /obj/item/clothing/under/syndicate
 	suit = /obj/item/clothing/suit/armor/vest/ctf
-	toggle_helmet = FALSE // see the whites of their eyes
```

Comment:
I have confirmed with thunder that this bit shouldn't be here anymore so its safe to remove.